src/core/player.js

```diff
@@ -21,33 +24,51 @@ export class TankPlayer {
         return this.playerY;
     }
 
-    async createBullet() {
-        const bullet = new BulletProjectile(this.playerX, this.playerY, this.app);
-        await bullet.initialiseBulletSprite();
-        this.app.stage.addChild(bullet.getSprite());
-        this.addBulletToBullets(bullet);
+    checkIfBulletIsPresent() {
+        if (this.bullets.length == 1) {
```

Comment:
For speed can't we just check a true false here instead of having to measure length too?
What I mean is wouldn't this.bullets be false? this.bullets should never exceed 1 right?